lunatik.h

```diff
@@ -147,10 +128,21 @@ static inline void *lunatik_checknull(lua_State *L, void *ptr)
 
 #define lunatik_checkalloc(L, s)	(lunatik_checknull((L), lunatik_malloc((L), (s))))
 
+static inline const char *lunatik_errname(int err)
+{
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 7, 0)
+	return errname(err);
+#else
+	static char buf[32];
+	snprintf(buf, sizeof(buf), "%pE", ERR_PTR(-err));
```

Comment:
I would do this directly in the lualinux_errname binding.. pushing it to the lua stack, where it will be copied instead of using a static (Oo) var for this.